en/docs/design-the-project/explore-the-architecture-view.md

```diff
@@ -1,10 +1,8 @@
-# Architecture Diagram
+# Explore the Architecture View
 
-Software applications have evolved into a composition of multiple interrelated components to be able to handle increasing complexities. When the number and/or depth of such components grow, it is convenient to have a high-level picture of the system components and their interactions. 
```

Comment:
Why this is removed?

Reply:
Added this to the overview page of `Design the project`.